numerrin-interface/libpcore.cpp

```diff
@@ -0,0 +1,1041 @@
+#include <sstream>
```

Comment:
Thanks for the quick response.  I have simphony-nummerin wrapper running now on my machine. :+1: 

>currently there is only precompiled libraries. To get things work you should have the path of libraries in LD_ LIBRARY_PATH. Library files are in numerrin-interface directory. I think that all proprietary codes should be distributed as precompiled libraries to avoid licensing issues.

Having "libnumerrin4.so"  in LD_LIBRARY_PATH worked but I needed to place "numerrin.so" in the pythons module search path (for me that was `~/.local/lib/python2.7/site-packages/.`). 

> libpcore.cpp is in the package while these codes were planned to be under GPL

Okay, I understand now that nummerrin.so is built from libpcore.cpp.  Maybe, it would have only the source file in the repository (libpcore.cpp) and the proper setup/scripts created to make and install the library (libpcore.cpp .. `numerrin` module) .   **I would suggest creating an github issue so that this can be addressed.**